ZeldaFullEditor/ZSCustomOverworld.asm

```diff
@@ -1069,25 +1069,27 @@ pushpc
 
 if !Func00EEBB == 1
 
-; Zeros out the BG color when mirror warping to the pyramid area.
-; TODO: This is done in the vanilla I think as just a precaution at the apex of
-; the fade to white to make sure all of the colors truly are white but it may
-; not actually be needed.
+; Zeros out the BG color when mirror warping from an area with the pyramid BG.
+; This is done to prevent a case where the black transparent color is faded to white
+; on top of the pyramid BG, resulting in a double faded effect on transparent tiles.
```

Comment:
RestoreBgColorFromParallaxBgMap for a name? I think parallax is probably the right term? with the way it sits in the background

Reply:
maybe it would have been but I was wrong with my original assumption and the understanding of its use is entirely different than it was before lol.